docs/examples.md

```diff
@@ -420,6 +420,27 @@ object PerftConverter extends IOApp.Simple {
 ```
 @:@
 
+## A helper utility to write out an `Seq[CaseClass]` to CSV
+
+In the presence of a CsvRowEncoder
+
+```
+import fs2.*
+import fs2.data.csv.*
+import fs2.io.file.Path
+
+given val pEncoder: CsvRowEncoder[CaseClass, String] = deriveCsvRowEncoder
+
+def writeCaseClassIterableToFile[A](data: Seq[A], file : Path)(using CsvRowEncoder[A, String]) =
+    fs2.Stream.emits[IO, A](data)
+      .through(encodeUsingFirstHeaders(fullRows = true))
+      .through(text.utf8Encode)
+      .through( Files[IO].writeAll(file) )
+      .compile
+      .drain      
```

Comment:
I wonder if this would be more idiomatic as a pipe? Then it doesn't force you to have all your data in-memory, instead you can stream it.

```suggestion
def writeCaseClassToCsv[A](path: Path)(using CsvRowEncoder[A, String]): Pipe[IO, A, Nothing] =
    _.through(encodeUsingFirstHeaders(fullRows = true))
      .through(text.utf8Encode)
      .through(Files[IO].writeAll(path))
      .drain      
```